gdeploylib/global_vars.py

```diff
@@ -31,7 +31,15 @@ class Global(object):
     var_file = None
     master = None
     brick_hosts = []
-    log_file = os.path.expanduser('~/.gdeploy/logs/gdeploy.log')
```

Comment:
I am trying to keep away as much as logic as possible from global_vars. global_vars can be used as an easy lookup for all the global variables. That is the idea. Having this huge logic in global bars kind of beat the purpose. Also, logging is altogether an independent framework as per the current architecture. This would mess it up.